documentation/assemblies/assembly-installing-configuring-olm.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-installing.adoc
+// Installing master.adoc
+// Evaluating master.adoc
+
+:parent-context: {context}
+
+[id='assembly-installing-configuring-olm-{context}']
+= Installing and configuring {ProductName} using the Operator Lifecycle Manager
+
+You can use the Operator Lifecycle Manager to install and configure an instance of {ProductName}.
+
+In OpenShift Container Platform 4.1, the Operator Lifecycle Manager (OLM) helps users install, update, and  manage the life cycle of all Operators and their associated services running across their clusters. It is part of the Operator Framework, an open source toolkit designed to manage Kubernetes native applications (Operators) in an effective, automated, and scalable way.
```

Comment:
Replace all occurences of "OpenShift Container Platform" with {KubePlatform}. In upstream docs, it should be just 'OpenShift'. Consider using 4.x or 4 instead of 4.1, because this applies to 4.2 as well.